authbridge/demos/aiac-github-issue/setup_keycloak.py

```
@@ -0,0 +1,843 @@
+        assigned: List[str] = []
+        for role in client_info["roles"]:
+            # Extract role name - support both dict and string formats
+            role_name = role["name"] if isinstance(role, dict) else role
+            scope_id = scope_ids.get(role_name)
+            if not scope_id:
+                continue
+            try:
+                admin.add_client_default_client_scope(client_info["id"], scope_id, {})
+                assigned.append(role_name)
+            except Exception:
+                pass  # Already added
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)